roles/openshift_version/tasks/main.yml

```diff
@@ -1,206 +1,2 @@
-    include_tasks: set_version_containerized.yml
-    when: openshift_is_containerized | bool
-
-  - block:
-    - name: Get available {{ openshift_service_type}} version
-      repoquery:
-        name: "{{ openshift_service_type}}"
-        ignore_excluders: true
-      register: rpm_results
-    - fail:
-        msg: "Package {{ openshift_service_type}} not found"
-      when: not rpm_results.results.package_found
-    - set_fact:
-        openshift_rpm_version: "{{ rpm_results.results.versions.available_versions.0 | default('0.0', True) }}"
-    - name: Fail if rpm version and docker image version are different
-      fail:
-        msg: "OCP rpm version {{ openshift_rpm_version }} is different from OCP image version {{ openshift_version }}"
-      # Both versions have the same string representation
-      when:
-      - openshift_rpm_version != openshift_version
-      # if openshift_pkg_version or openshift_image_tag is defined, user gives a permission the rpm and docker image versions can differ
-      - openshift_pkg_version is not defined
-      - openshift_image_tag is not defined
-    when:
-    - openshift_is_containerized | bool
-    - not openshift_is_atomic | bool
-
-  # Warn if the user has provided an openshift_image_tag but is not doing a containerized install
-  # NOTE: This will need to be modified/removed for future container + rpm installations work.
-  - name: Warn if openshift_image_tag is defined when not doing a containerized install
-    debug:
-      msg: >
-        openshift_image_tag is used for containerized installs. If you are trying to
-        specify an image for a non-container install see oreg_url or oreg_url_master or oreg_url_node.
-    when:
-    - not openshift_is_containerized | bool
-    - openshift_image_tag is defined
-
-  # At this point we know openshift_version is set appropriately. Now we set
-  # openshift_image_tag and openshift_pkg_version, so all roles can always assume
-  # each of this variables *will* be set correctly and can use them per their
-  # intended purpose.
-
-  - block:
-    - debug:
-        msg: "openshift_image_tag was not defined. Falling back to v{{ openshift_version }}"
-
-    - set_fact:
-        openshift_image_tag: v{{ openshift_version }}
-
-    when: openshift_image_tag is not defined
-
-  - block:
-    - debug:
-        msg: "openshift_pkg_version was not defined. Falling back to -{{ openshift_version }}"
-
-    - set_fact:
-        openshift_pkg_version: -{{ openshift_version }}
-
-    when:
-    - openshift_pkg_version is not defined
-    - openshift_upgrade_target is not defined
-
-  - fail:
-      msg: openshift_version role was unable to set openshift_version
-    name: Abort if openshift_version was not set
-    when: openshift_version is not defined
-
-  - fail:
-      msg: openshift_version role was unable to set openshift_image_tag
-    name: Abort if openshift_image_tag was not set
-    when: openshift_image_tag is not defined
-
-  - fail:
```

Comment:
This check was eliminated due to you won't be able to ever meet these conditions now.